ESSArch_TA/frontend/static/frontend/gulpfile.js

```diff
@@ -285,15 +296,16 @@ gulp.task('core_tests', buildCoreTests);
 gulp.task('scripts', buildScripts);
 gulp.task('vendors', buildVendors);
 gulp.task('sass', compileSass);
+gulp.task('rev', buildRevManifest);
 gulp.task('config', configConstants);
 gulp.task('permission_config', permissionConfig);
 
 gulp.task('watch', function(){
-    gulp.watch(coreHtmlFiles, ['core_templates']);
-    gulp.watch(coreJsFiles, ['core_scripts']);
-    gulp.watch(coreTestFiles, ['core_tests']);
-    gulp.watch(jsFiles, ['scripts']);
-    gulp.watch(jsPolyfillFiles, ['polyfills']);
-    gulp.watch(vendorFiles, ['vendors']);
-    gulp.watch(cssFiles, ['sass']);
+    gulp.watch(coreHtmlFiles, function(){runSequence('core_templates', 'rev')});
+    gulp.watch(coreJsFiles, function(){runSequence('core_scripts', 'rev')});
+    gulp.watch(coreTestFiles, function(){runSequence('core_tests', 'rev')});
+    gulp.watch(jsFiles, function(){runSequence('scripts', 'rev')});
+    gulp.watch(jsPolyfillFiles, function(){runSequence('polyfills', 'rev')});
+    gulp.watch(jsVendorFiles, function(){runSequence('vendors', 'rev')});
```

Comment:
jsVendorFiles doesn't match the variable above. Either change that or this variable